developer/src/kmc-analyze/test/osk-character-use-warnings.tests.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ */
+import * as fs from 'fs';
+import { assert } from 'chai';
+import 'mocha';
+import { AnalyzeOskCharacterUse } from '../src/osk-character-use/index.js';
+import { TestCompilerCallbacks } from '@keymanapp/developer-test-helpers';
+import { AnalyzerMessages } from '../src/analyzer-messages.js';
+
+describe('AnalyzeOskCharacterUse warnings', function() {
+    const callbacks = new TestCompilerCallbacks();
```

Comment:
Nit: We use [two spaces](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#Code-Format) for indents. (I think your previous PR added to an existing file so indents matched.)